### ISSUE TITLE
Getting ERR_PACKAGE_PATH_NOT_EXPORTED

### ISSUE THREAD

**Issue by workergnome** [jaydenseric/jsdoc-md#24]
On a new install, when I try and run this using npx: 

`npx jsdoc-md`

It gives me the following output:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/parser/index.js' is not defined by "exports" in /my/path/node_modules/comment-parser/package.json imported from /my/path/node_modules/jsdoc-md/private/jsdocCommentToMember.mjs
```

Any ideas?  Could very well be user error on my part.

**Comment by jaydenseric** [comment]
`comment-parser` published major breaking changes without a major version bump, see:

https://github.com/syavorsky/comment-parser/commit/ac45b9b69487dc6760239d881339da18581d5531#commitcomment-54215184

I'll fix this here in a new version in a few hours (just waking up right now).

**Comment by workergnome** [comment]
Fabulous!  Thank you for looking into it.

**Comment by jaydenseric** [comment]
Although the `comment-parser` guys still haven't confirmed the game plan, I've published a fix in [v11.0.1](https://github.com/jaydenseric/jsdoc-md/releases/tag/v11.0.1). Thanks for your patience!

**Comment by workergnome** [comment]
Fabulous.  Can confirm the new version works.

Thank you so much!
